gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStorageStatistics.java

```diff
@@ -16,16 +16,88 @@
 
 package com.google.cloud.hadoop.fs.gcs;
 
+import java.util.Map;
 import org.apache.hadoop.fs.statistics.IOStatistics;
+import org.apache.hadoop.fs.statistics.MeanStatistic;
+import org.apache.hadoop.fs.statistics.StoreStatisticNames;
 import org.apache.hadoop.fs.statistics.impl.StorageStatisticsFromIOStatistics;
 
 /** Storage statistics for GCS, dynamically generated from the IOStatistics. */
 public class GhfsStorageStatistics extends StorageStatisticsFromIOStatistics {
 
+  /** {@value} The key that stores all the registered metrics */
   public static final String NAME = "GhfsStorageStatistics";
 
+  /** Exention for minimum */
+  private static final String MINIMUM = StoreStatisticNames.SUFFIX_MIN;
+  /** Exention for maximum */
+  private static final String MAXIMUM = StoreStatisticNames.SUFFIX_MAX;
+  /** Exention for mean */
+  private static final String MEAN = StoreStatisticNames.SUFFIX_MEAN;
+
+  /** IOStatistics Instance */
+  private final IOStatistics ioStatistics;
+
   /** Create the Storage Statistics instance from the IOStatistics */
   public GhfsStorageStatistics(IOStatistics ioStatistics) {
     super(NAME, "Ghfs", ioStatistics);
+    this.ioStatistics = ioStatistics;
+  }
+
+  /**
+   * Map of minimums
+   *
+   * @return current map of minimums
+   */
+  private Map<String, Long> minimums() {
```

Comment:
Do any of these `private` functions adding any value?

Reply:
We could have inlined this. Having this method is not bad either.